src/WebJobs.Script/Workers/IWorkerChannel.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Workers
+{
+    public interface IWorkerChannel
+    {
+        string Id { get; }
```

Comment:
Are there any other common methods all channels have that should be included in this interface, e.g. StartWorkerProcessAsync?

Reply:
No

`StartWorkerProcessAsync `has different signature:
```
IRpcWorkerChannel -  Task StartWorkerProcessAsync();
IHttpWorkerChannel -  Task StartWorkerProcessAsync(CancellationToken cancellationToken = default);
```

Reply:
We should be able to just converge on the second signature taking the cancellation token